eos-add-flatpak-gnome-repos

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+# Copyright (C) 2016 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+# Add the stable GNOME flatpak repositories after an OS upgrade
+
+TMPDIR=`mktemp -d`
+
+# Sanity check
+if [ -z ${TMPDIR} ] || [ ! -d ${TMPDIR} ] ; then
+    echo "Error creating temporary directory, exiting"
+    exit 0
+fi
```

Comment:
This block is probably not needed since you're running with `-e` and bash will exit immediately if the `mktemp` fails. However, if you do want to keep the sanity check, I'd change this to `exit 1` so it shows up as a failure if the temporary directory wasn't created.